odl/tomo/geometry/spect.py

```diff
@@ -0,0 +1,82 @@
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SPECT geometry."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+import numpy as np
+
+from odl.tomo.geometry.geometry import AxisOrientedGeometry
+from odl.tomo.geometry.parallel import Parallel3dAxisGeometry
+
+__all__ = ('ParallelHoleCollimatorGeometry', )
+
+
+class ParallelHoleCollimatorGeometry(Parallel3dAxisGeometry):
+
+    """Geometry for SPECT Parallel hole collimator."""
+
+    def __init__(self, apart, dpart, axis=[0, 0, 1], det_rad=0.0,
+                 **kwargs):
+        """Initialize a new instance.
+        Parameters
+        ----------
+        apart : 1-dim. `RectPartition`
+            Partition of the angle interval
+        dpart : 2-dim. `RectPartition`
+            Partition of the detector parameter rectangle
+        axis : `array-like`, shape ``(3,)``, optional
+            Fixed rotation axis.
+        det_rad : nonnegative `float`, optional
+            Radius of the circular acquisition curve
+        orig_to_det_init : `array-like`, shape ``(3,)``, optional
+            Vector pointing towards the detector reference point in
+            the initial position. The zero vector is only allowed if
+            ``det_init_axes`` is explicitly given.
+            By default, a `perpendicular_vector` to ``axis`` is used.
+        det_init_axes : 2-tuple of `array-like` (shape ``(3,)``), optional
+            Initial axes defining the detector orientation.
+            By default, the normalized cross product of ``axis`` and
+            ``det_init_pos`` is used as first axis and ``axis`` as second.
+        """
+        # Use det_init_pos to initialize the detector position, then rescale
+        orig_to_det_init = kwargs.pop('orig_to_det_init', None)
+        if orig_to_det_init is not None:
+            kwargs['det_init_pos'] = orig_to_det_init
+        super().__init__(apart, dpart, axis, **kwargs)
+        self._det_radius = float(det_rad)
+        if self.det_radius < 0:
+            raise ValueError('expected a nonnegative radius, got {}.'
+                             ''.format(det_rad))
+
+        # Make sure that the initial position lies on the specified circle
+        init_pos_norm = np.linalg.norm(self._det_init_pos)
+        if init_pos_norm > 1e-10:
+            self._det_init_pos *= self.det_radius / init_pos_norm
```

Comment:
Second thought, we are chaging the meaning of this guy here. In the parent class, this is a vector to the detector reference point. Here you change it to mean a unit vector _towards_ the detector reference point. This will cause bugs, try calling `det_refpoint` and you'll see.